conf_linux_public.py

```diff
@@ -139,28 +177,43 @@ def check_lib_size(threshold_size, lib_path):
     raise IOError(f"Unknown product type '{product_type}'")
 
 
-PRODUCT_REPOS = [
-    {'name': repo_name},
-    # Give possibility to build linux for changes from product configs repository
-    # This repo not needed for build and added only to support CI process
-    {'name': 'product-configs'}
-    #{'name': 'flow_test'},
-]
+action('count api version and build number',
+       callfunc=(set_env, [MEDIA_SDK_REPO_DIR, GCC_LATEST, CLANG_VERSION], {}))
 
-ENABLE_DEVTOOLSET = 'source /opt/rh/devtoolset-6/enable'
-GCC_LATEST = '8.2.0'
-CLANG_VERSION = '6.0'
-options["STRIP_BINARIES"] = True
-MEDIA_SDK_REPO_DIR = options.get('REPOS_DIR') / repo_name
+# Build dependencies
+# Build LibVA
+# prefix = /usr/local
+action('LibVA: autogen.sh',
+       work_dir=libva_options['BUILD_DIR'],
+       cmd=get_building_cmd(f'{LIBVA_REPO_DIR}/autogen.sh', GCC_LATEST, ENABLE_DEVTOOLSET))
 
+action('LibVA: build',
+       work_dir=libva_options['BUILD_DIR'],
+       cmd=get_building_cmd(f'make -j`nproc`', GCC_LATEST, ENABLE_DEVTOOLSET))
 
-action('count api version and build number',
-       callfunc=(set_env, [MEDIA_SDK_REPO_DIR, GCC_LATEST, CLANG_VERSION], {}))
+action('LibVA: list artifacts',
+       work_dir=libva_options['BUILD_DIR'],
+       cmd=f'echo " " && ls ./va',
+       verbose=True)
+
+# libva should be installed before MediaSDK build
+# install on the build stage
+action('install libva',
```

Comment:
please use LibVA everywhere in messages
should be like: LibVA: make install
it is good to use the LibVA prefix for all actions with LibVA